packages/messenger-widget/src/hooks/conversation/useConversation.tsx

```diff
@@ -209,12 +209,22 @@ export const useConversation = (config: DM3Configuration) => {
             DEFAULT_CONVERSATION_PAGE_SIZE,
             Math.floor(offset),
         );
+
+        let newContactAddedCount = 0;
```

Comment:
Please return conversation.length instead of counting it separatly

Reply:
Here if somehow our own account address is found, then it is not added in contact list. But it will be counted in conversations.length. So to avoid that count is used

Reply:
The account cannot by part of the contact list at this part because it is not possible to add it in the first place

Reply:
So we can ignore it here

Reply:
okay